lib/dogapi/facade.rb

```diff
@@ -296,6 +297,29 @@ def graph_snapshot(metric_query, start_ts, end_ts, event_query = nil)
     end
 
     #
+    # EMBEDS
+    #
+    def get_all_embeds()
+      @embed_svc.get_all_embeds()
+    end
+
+    def get_embed(embed_id, size="medium", legend="no", template_vars= {})
+      @embed_svc.get_embed(embed_id, size, legend, template_vars)
+    end
+
+    def create_embed(graph_json, timeframe="1_hour", size="medium", legend="no", title="Embed created through API")
```

Comment:
It isn't required. Does I make title required in the Ruby code? I'm not super familiar with Ruby so I assumed the default parameters work the same way as they do in Python